packages/integrations/cloudflare/test/test-utils.js

```diff
@@ -21,20 +22,30 @@ const wranglerPath = fileURLToPath(
 );
 
 /**
- * @returns {WranglerCLI}
+ * @returns {Promise<WranglerCLI>}
  */
-export function runCLI(basePath, { silent, port = 8787 }) {
+export async function runCLI(basePath, { silent, port }) {
+	// Hack: force existing process on port to be killed
+	try {
+		await kill(port, 'tcp');
+	} catch {
+		// Will throw if port is not in use, but that's fine
+	}
```

Comment:
This was needed because apparently `wrangler` didn't always exit cleanly

Reply:
Nice fix. I also tried updating wrangler the last time but was stuck with this. Strange that wrangler doesn't properly teardown its server sometimes.